lib/Monocle.php

```diff
@@ -31,6 +31,15 @@ public function get_active($eids, $minIv, $minLevel, $exMinIv, $swLat, $swLng, $
             $params[':oneLng'] = $oNeLng;
         }
         if (count($eids)) {
+            $tmpSQL = '';
+            if(!empty($tinyRat) && $tinyRat === 'true' && ($key = array_search("19", $eids)) === false){
```

Comment:
Would `in_array` be more appropriate here? I think it's marginally quicker if all you need to do is verify if something is in an array?